boot/zephyr/Kconfig

```diff
@@ -305,6 +305,7 @@ config BOOT_SERIAL_DETECT_PIN
 	default 6 if BOARD_NRF9160_PCA10090
 	default 11 if BOARD_NRF52840_PCA10056
 	default 13 if BOARD_NRF52_PCA10040
+	default 23 if BOARD_NRF5340_DK_NRF5340_CPUAPP || BOARD_NRF5340_DK_NRF5340_CPUAPPNS
```

Comment:
strange that same pin is used for both

Reply:
It is only the non-secure alias for the same SoC, hence the same pin (it is button 1 on the 53 DK)

Reply:
good, thanks for clarification